samples/bluetooth/direct_test_mode/overlay-hci-nrf53.conf

```diff
@@ -0,0 +1,15 @@
+#
+# Copyright (c) 2023 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+CONFIG_NCS_SAMPLE_REMOTE_SHELL_CHILD_IMAGE=n
+CONFIG_IPC_UART=n
+
+CONFIG_NCS_SAMPLE_DTM_REMOTE_HCI_CHILD_IMAGE=y
```

Comment:
Where is this image and how it is added to project?

Reply:
It is in the project directory in `remote_hci` directory. The image is added the same way as other images - the CMake adds it if Kconfig is correct - in this case `CONFIG_NCS_SAMPLE_DTM_REMOTE_HCI_CHILD_IMAGE`.